github-get-reviewers.go

```diff
@@ -16,14 +16,10 @@
 
 package main
 
-import "fmt"
-
-// ReviewStatus - holds Review info
-type ReviewStatus struct {
-	User struct {
-		Login string `json:"login"`
-	} `json:"user"`
-}
+import (
+	"github.com/google/go-github/github"
```

Comment:
Restructure the import here.. follow this style https://github.com/golang/go/wiki/CodeReviewComments#imports